airflow/providers/microsoft/psrp/operators/psrp.py

```diff
@@ -56,8 +56,11 @@ class PsrpOperator(BaseOperator):
     :param cmdlet:
         cmdlet to execute on remote host (templated). Also used as the default
         value for `task_id`.
+    :param arguments:
+        When using the `cmdlet` or `powershell` option, use `arguments` to
+        provide arguments (templated).
     :param parameters:
-        When using the `cmdlet` or `powershell` arguments, use this parameter to
+        When using the `cmdlet` or `powershell` option, use `parameters` to
         provide parameters (templated). Note that a parameter with a value of `None`
         becomes an *argument* (i.e., switch).
```

Comment:
I’m assuming users fluent with Powershell would understand the difference between these? I wonder if we should add a link to some documentation about this.

Reply:
There's a link in the provider documentation to https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-psrp/.

Most Windows superusers will be familiar with the [Invoke-Command](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/invoke-command) cmdlet which has the same interface.